osu.Game.Rulesets.Osu/OsuDifficulty/Preprocessing/OsuDifficultyHitObject.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright (c) 2007-2017 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using System;
+using osu.Game.Rulesets.Osu.Objects;
+
+namespace osu.Game.Rulesets.Osu.OsuDifficulty.Preprocessing
+{
+    public class OsuDifficultyHitObject

```

Comment:
The class itself should also have a description of what is does. I.e. mention that `OsuDifficultyHitObject` contains / derives _extra_ information about `BaseObject` which is required for difficulty calculations.